src/current/v23.2/show-ranges.md

```diff
@@ -77,6 +77,36 @@ Field | Description | Emitted for option(s)
 `index_end_key` | The end key of the last [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) of [index]({% link {{ page.version.version }}/indexes.md %}) data. | `INDEXES`
 `raw_index_start_key` | The start key of the first [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) of [index]({% link {{ page.version.version }}/indexes.md %}) data, expressed as [`BYTES`]({% link {{ page.version.version }}/bytes.md %}). | `INDEXES`, `KEYS`
 `raw_index_end_key` | The end key of the last [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) of [index]({% link {{ page.version.version }}/indexes.md %}) data, expressed as [`BYTES`]({% link {{ page.version.version }}/bytes.md %}). | `INDEXES`, `KEYS`
+`span_stats` | A `JSON` object containing span statistics. For more details, see [Span Statistics](#span-statistics). | `DETAILS`
+
+### Span Statistics
+
+CockroachDB stores all user data in a sorted map of key-value pairs, also known as a keyspace. A span refers to an interval within this keyspace.
+
+{% include_cached new-in.html version="v23.2" %} The `SHOW RANGES` command emits span statistics when the `DETAILS` option is specified. The statistics are included in a column named `span_stats`, as a `JSON` object.
+
+The statistics are calculated for the identifier of each row. For example,
+
+- `SHOW RANGES WITH DETAILS` will compute span statistics for each [range]({% link {{ page.version.version }}/ui-replication-dashboard.md %}#review-of-cockroachdb-terminology).
+- `SHOW RANGES WITH TABLES, DETAILS` will compute span statistics for each table, and so on.
+ 
+The `span_stats` `JSON` object has the following keys:
+
+- `approximate_disk_bytes`
+- `[key|val|sys|live|intent]_count`
+- `[key|val|sys|live|intent]_bytes`
+
+`approximate_disk_bytes` is an approximation of the total on-disk size of the given object, across all [replicas]({% link {{ page.version.version }}/architecture/glossary.md %}#replica) (after compression).
```

Comment:
I find this section a little hard to read — I think it's because we're discussing separate items that are formatted in body text. Also, some lines have two keys per line (I understand they are connected or rely on one another).

Maybe these could be in a table with key and description as the columns? Similar to the Parameters table a little way above?